### ISSUE TITLE
Message download not working

### ISSUE THREAD

**Issue by steffenfritz** [DavidMStraub/netviel#18]
When I try to download a message an internal server error occurs.

**Stacktrace:**

```
Error on request:
Traceback (most recent call last):
  File "/home/steffen/.local/lib/python3.9/site-packages/werkzeug/serving.py", line 335, in run_wsgi
    execute(self.server.app)
  File "/home/steffen/.local/lib/python3.9/site-packages/werkzeug/serving.py", line 322, in execute
    application_iter = app(environ, start_response)
  File "/home/steffen/.local/lib/python3.9/site-packages/flask/app.py", line 2548, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/steffen/.local/lib/python3.9/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.handle_exception(e)
  File "/home/steffen/.local/lib/python3.9/site-packages/flask_restful/__init__.py", line 271, in error_router
    return original_handler(e)
  File "/home/steffen/.local/lib/python3.9/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/steffen/.local/lib/python3.9/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/steffen/.local/lib/python3.9/site-packages/flask_restful/__init__.py", line 271, in error_router
    return original_handler(e)
  File "/home/steffen/.local/lib/python3.9/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/steffen/.local/lib/python3.9/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/usr/home/steffen/Code/netviel/netviel/api.py", line 130, in download_message
    return send_file(msg.get_filename(), mimetype="message/rfc822",
TypeError: send_file() got an unexpected keyword argument 'attachment_filename'
```

**Reason:**

https://github.com/pallets/flask/issues/4753

**tl;dr:** 

download_name replaces attachment_filename
